packages/web-wallet/src/state/editDapp/selectors.js

```diff
@@ -0,0 +1,24 @@
+import { createSelector } from 'reselect';
+import { isNil, isEmpty } from 'lodash';
+
+import { getUserAccountId } from 'state/userAccounts';
+import { getDappCatalog } from 'state/storeAccount';
+
+export const getEditStep = state => state.editDapp.editStep;
+
+export const getIsDappSubmitted = createSelector(
+  [getUserAccountId, getDappCatalog],
+  (userAccountId, dappCatalog) => {
+    if (isEmpty(dappCatalog)) {
+      return false;
+    }
+
+    const entry = dappCatalog[userAccountId];
+
+    if (isNil(entry)) {
+      return false;
+    }
+
+    return true;
```

Comment:
Two if statements may be replaced with one: `return isNil(get(dappCatalog, 'userAccountId'))`

Reply:
I would prefer to make things as simple and obvious as possible and do not chain boolean checking with object value getter.